### ISSUE TITLE
bold smart lock entity unavailable

### ISSUE THREAD

**Issue by yslaston** [lwestenberg/homeassistant_bold#31]
Hello, hopefully somebody could bring light on my problem. I was able to install bold smart lock via HACS on home assistant, running on virtual machine. Integration/authorization via nabu casa cloud  was also successfull. Unfortunatelly after many tries and re-installs  the lock entity is greyed out with status unavailble. Any suggestions are higly appreciated.
![bold unavailablle](https://github.com/lwestenberg/homeassistant_bold/assets/171819253/48600c1d-1688-4051-9547-bb9b25b1a028)

**Comment by yslaston** [comment]
this is what I found in logs
![image](https://github.com/lwestenberg/homeassistant_bold/assets/171819253/529dbcd6-261c-4594-866c-ff618bc264f6)

**Comment by tim427** [comment]
Somehow your HomeAssistant thing there is already a Lock Entity on the Bold platform with device_id "121401".

Potentially some leftovers from earlier attempts?

Try to delete all Bold entities, stop HomeAssistant and verify if "121401" still exists in `.storage/core.entity_registry` and/ord `.storage/core.device_registry`

**Comment by yslaston** [comment]
Thanks so far... as I'm a newbie in home assitant, how can I access registry with entities and devices? SSH terminal?

**Comment by yslaston** [comment]
![image](https://github.com/lwestenberg/homeassistant_bold/assets/171819253/7d6bef81-f521-483e-bf0e-379adf29dff3)

**Comment by yslaston** [comment]
Well, finally I was able to access those folders and deleted rows in entitity and device registry. I also restarted few times home assistant. Added lock once again. It didn't help! You guys are doing really great job. Honestly! But, unfortunatelly because of issue like mine, home assistant will never become a popular smart home system among usual people without strong IT background, who just want a stable running system. Good luck!
